chrome/background.js

```diff
@@ -21,6 +24,23 @@ function fillLoginForm(login) {
       });
     }
   );
+
+  if (login.digits) {
+    chrome.tabs.query({ active: true }, function(tabs) {
```

Comment:
this is already queried in `onMessage` where `fillLoginForm` is called, maybe just pass the tabID from there and don't query again?